.github/FUNDING.yml

```
@@ -0,0 +1,3 @@
+# These are supported funding model platforms
+
+github: [eddiejaoude]
```

Missing blank line ending

```suggestion
github: [eddiejaoude]

```